storage/google/cloud/storage/client.py

```diff
@@ -309,6 +311,31 @@ def create_bucket(self, bucket_or_name, requester_pays=None, project=None):
         bucket.create(client=self, project=project)
         return bucket
 
+    def download_blob_to_file(self, blob_or_uri, file_obj, start=None, end=None):
+        """Download the contents of a blob object or blob URI into a file-like object.
+
+        Args:
+            blob_or_uri (Union[ \
+                :class:`~google.cloud.storage.blob.Blob`, \
+                 str, \
```

Comment:
Nit: This doesn't quite line up with the line above it (there's one extra space here)

Reply:
Fixed!